zerver/lib/i18n.py

```diff
@@ -22,16 +22,10 @@ def with_language(string, language):
 
```

Comment:
Should we also move this to `compilemessages.py`?  I think at least we should add a note suggesting that this is an expensive operation and shouldn't be done often.

Reply:
I have added a comment in that function. It wasn't feeling right to place it in `compilemessages.py` :/.